### PR TITLE
feature: improve secrets cmd

### LINKED ISSUE #3157: Improve the `secrets` command
The goal of this issue is to improve the [secret](https://github.com/nextflow-io/nextflow/blob/27636f9d27c56d286e1e6173e66d97c971c5d6d3/modules/nextflow/src/main/groovy/nextflow/cli/CmdSecret.groovy#L52-L56) command as describe below

- `secret get <NAME>` (instead of using `-n` option) 
- `secret put <NAME> <VALUE>` (instead of using `-n` and `-v` options)
- `secret delete <NAME>` (instead of using the `-n` option)

### PR DESCRIPTION
**Pull request by jorgeaguileraseqera**
closes #3157 

Signed-off-by: Jorge Aguilera <jorge.aguilera@seqera.io>

### CODE REVIEW

File: modules/nextflow/src/main/groovy/nextflow/cli/CmdSecret.groovy [hunk 1]
```diff
@@ -99,7 +95,7 @@ class CmdSecret extends CmdBase implements UsageAware {
             }
         }
 
-        println result.join('\n').toString()
+        out.println result.join('\n').toString()
```
Comment (pditommaso) [inline modules/nextflow/src/main/groovy/nextflow/cli/CmdSecret.groovy:98]: Why `out.` ?

Comment (jorgeaguileraseqera) [reply modules/nextflow/src/main/groovy/nextflow/cli/CmdSecret.groovy:98]: I've used the same "trick" in CmdInfo to read the output

Comment (pditommaso) [reply modules/nextflow/src/main/groovy/nextflow/cli/CmdSecret.groovy:98]: Ohh, there's a [better trick](https://github.com/nextflow-io/nextflow/blob/27636f9d27c56d286e1e6173e66d97c971c5d6d3/modules/nextflow/src/test/groovy/nextflow/cli/CmdLogTest.groovy#L41-L45) !

Comment (jorgeaguileraseqera) [reply modules/nextflow/src/main/groovy/nextflow/cli/CmdSecret.groovy]: changed

**Comment by pditommaso** [comment]
Nice, thanks

**Pull request merged**